src/pages/RedeemPosition/Contents.tsx

```diff
@@ -24,12 +25,42 @@ const logger = getLogger('RedeemPosition')
 
 export const Contents = () => {
   const { _type: status, CTService, connect, networkConfig } = useWeb3ConnectedOrInfura()
-  const { clearPositions, errors: positionsErrors, positions } = useMultiPositionsContext()
+  const {
+    clearPositions,
+    errors: positionsErrors,
+    loading: loadingPositions,
+    positions,
+  } = useMultiPositionsContext()
+  const {
+    errors: balancesErrors,
+    loading: loadingBalance,
+    updateBalances,
+  } = useBatchBalanceContext()
 
-  const { clearCondition, condition, errors: conditionErrors } = useConditionContext()
+  const {
+    clearCondition,
+    condition,
+    errors: conditionErrors,
+    loading: loadingCondition,
+  } = useConditionContext()
   const [statusTransaction, setStatusTransaction] = React.useState<Maybe<Status>>(null)
   const [error, setError] = React.useState<Maybe<Error>>(null)
 
+  useEffect(() => {
+    if (positions) {
+      updateBalances(positions.map((p) => p.id))
+    }
+  }, [positions, updateBalances])
```

Comment:
`updateBalances` from this component, to also have available the balance errors as part of the logic of disabling the redeem button.